src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseAnalyzers.cs

```diff
@@ -23,6 +23,37 @@ public GivenThatWeWantToUseAnalyzers(ITestOutputHelper log) : base(log)
         {
         }
 
+        [Theory]
+        [InlineData("WebApp", "EnableRequestDelegateGenerator", false)]
+        [InlineData("WebApp", "EnableRequestDelegateGenerator", true)]
+        public void It_resolves_requestdelegategenerator_correctly(string testAssetName, string property, bool isEnabled)
```

Comment:
```suggestion
        [InlineData("WebApp", false)]
        [InlineData("WebApp", true)]
        public void It_resolves_requestdelegategenerator_correctly(string testAssetName, bool isEnabled)
```

No need to pass a theory parameter that is always the same value, you can just hard-code this in the test body.